config.lua

```diff
@@ -9,6 +9,19 @@ Config.DurabilityBlockedWeapons = {
     "weapon_unarmed",
 }
 
+Config.Throwables = {
+    "weapon_snowball",
+    "weapon_pipebomb",
+    "weapon_molotov",
+    "weapon_stickybomb",
+    "weapon_grenade",
+    "weapon_bzgas",
+    "weapon_proxmine",
+    "weapon_ball",
+    "weapon_smokegrenade",
+    "weapon_flare"
```

Comment:
Since we're moving these, I wonder if it'd be nice to arrange these alphabetically for easy maintence 

`
    "weapon_ball",
    "weapon_bzgas",
    "weapon_flare"
    "weapon_grenade",
    "weapon_molotov",
    "weapon_pipebomb",
    "weapon_proxmine",
    "weapon_smokegrenade",
    "weapon_snowball",
    "weapon_stickybomb",
`

Reply:
Also. We have the option to omit the `weapon_` from this list and add it later  e.g.

```
    "ball",
    "bzgas",
    "flare",
    "grenade",
    "molotov",
    "pipebomb",
    "proxmine",
    "smokegrenade",
    "snowball",
    "stickybomb",
    ```